index.js

```diff
@@ -140,29 +144,11 @@ function onWorkerMessage (msg) {
       // Replace the FakeWeakRef with a
       // proper one.
       this.stream = new WeakRef(stream)
-      if (stream._sync) {
-        stream.ready = true
-        stream.flushSync()
```

Comment:
I'm probably lost on why this is not needed anymore.

Reply:
Because we don't wait for ready anymore before we start writing to the shared buffer, i.e. it's already flushed.

Reply:
Thanks!